index.js

```diff
@@ -18,6 +18,7 @@ const calculateCacheKeyForTree = require('calculate-cache-key-for-tree');
 const buildTree = require('./lib/broccoli/build-translation-tree');
```

Comment:
Issues I'm seeing right now testing this within my application:
* In my application I'm seeing translations duplicated in the payload - once in app.js and once in vendor.js.  I'd expect engine translations to be in vendor.js only.
* Engine's are unable to implement `treeForTranslations`